queries/dart/indents.scm

```diff
@@ -0,0 +1,21 @@
+[
+  (class_body)
+  (function_body)
+  (declaration (initializers))
+  (switch_block)
+  (if_statement)
+  (formal_parameter_list)
+  (list_literal)
```

Comment:
I think i'd add `formal_parameter` to this list ? i really have no idea how dart indenting works though